examples/purchaseTesterTypescript/ios/Podfile

```diff
@@ -1,7 +1,7 @@
 require_relative '../node_modules/react-native/scripts/react_native_pods'
 require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
 
-platform :ios, min_ios_version_supported
+platform :ios, "13.0"
```

Comment:
Currently, `min_ios_version_supported` of React Native is 12.4 but we only support 13+ in the new iOS SDK